README.md

```diff
@@ -15,4 +15,22 @@ make bundle-build
 podman push ${CONTAINER_REPO_OVERRIDE}/hyperfoil-operator-bundle:0.15.0
 # Creates CatalogSource and Subscription in the cluster
 operator-sdk run bundle ${CONTAINER_REPO_OVERRIDE}/hyperfoil-operator-bundle:0.15.0
-```
\ No newline at end of file
+```  
+
+## Deploy the operator and run Hyperfoil in minikube
+        
+First start by [installing minikube](https://minikube.sigs.k8s.io/docs/start/). Start the cluster with `m̀inikube start`. (*Suggestion:* use `minikube dashboard` to monitor the cluster)
+                       
+Build and install Hyperfoil operator (Go 1.19 is required) with `make build install`
+
+The example resource in [config/samples/_v1alpha2_hyperfoil.yaml](config/samples/_v1alpha2_hyperfoil.yaml) can be deployed with `make deploy-samples run`. 
+
+To expose the controller to the host, need to forward port 8090 with `kubectl port-forward --address 0.0.0.0 service/hyperfoil 8090:8090`
+
+You can connect to the Hyperfoil controller using the hyperfoil-cli or the [web-cli](http://127.0.0.1:8090)
+                                           
+From here on, follow the [Hyperfoil quickstart guide](https://hyperfoil.io/quickstart) for instructions on how to deploy and run a benchmark.
+
+### Undeploy the Hyperfoil operator
+
+`make undeploy-samples clean`
```

Comment:
We need a bit more here -- the `make deploy-samples run` is "blocked"; e.g. is it a separate windows, can you Ctrl-C, ...

Also, we need a `minikube stop` command